docs/resources/connection.md

```diff
@@ -263,13 +261,13 @@ With the `salesforce`, `salesforce-community` and `salesforce-sandbox` connectio
 
 ```hcl
 resource "auth0_connection" "salesforce" {
-	name = "Salesforce-Connection"
-	strategy = "salesforce"
-	options {
-		client_id = "<client-id>"
-		client_secret = "<client-secret>"
-		community_base_url = "https://salesforce.example.com"
-	}
+ name = "Salesforce-Connection"
+ strategy = "salesforce"
+ options {
+  client_id = "<client-id>"
+  client_secret = "<client-secret>"
+  community_base_url = "https://salesforce.example.com"
+ }
```

Comment:
Appreciate you opening a PR! But would rather us leave these indentations alone. I'm not even sure tabs are a "violation" and even if they were, for documentation purposes, the indentation they provide is more clear.